fastlane/actions/upload_dsym.rb

```diff
@@ -1,28 +1,28 @@
-require 'rest-client'
-
 module Fastlane
   module Actions
     class UploadDsymAction < Action
-      
-      # Given a dsym zip path, a Rollbar access token, 
+
+      # Given a dsym zip path, a Rollbar access token,
       # an app bundle identifier and an app version
-      # uploads the given dsym to Rollbar with 
+      # uploads the given dsym to Rollbar with
       # the provided configuration.
       # Returns if the upload was successful.
-      
+
       def self.run(params)
         # Uploads the zipped dsym file to Rollbar.
-        response = RestClient.post 'https://api.rollbar.com/api/1/dsym', { 
-          :access_token => params[:access_token],
-          :bundle_identifier => params[:bundle_identifier],
-          :version => params[:version],
-          :dsym => File.new(params[:dsym_zip_path]),
-          :multipart => true
-        }
-
-        # If upload was successful "error" is 0. 
+
+      response = %x<curl -X POST "https://api.rollbar.com/api/1/dsym" \
+          -F access_token=#{params[:access_token]} \
+          -F version=#{params[:version]} \
+          -F bundle_identifier=#{params[:bundle_identifier]} \
+          -F dsym=@"#{params[:dsym_zip_path]}"
+        >
```

Comment:
Wouldn't you move this to the upper line?

Reply:
Done!